dotnet/src/Connectors/Connectors.Memory.Pinecone/PineconeVectorStoreRecordCollection.cs

```diff
@@ -235,9 +239,67 @@ await this.RunOperationAsync(
     }
 
     /// <inheritdoc />
-    public Task<VectorSearchResults<TRecord>> VectorizedSearchAsync<TVector>(TVector vector, VectorSearchOptions? options = null, CancellationToken cancellationToken = default)
+    public async Task<VectorSearchResults<TRecord>> VectorizedSearchAsync<TVector>(TVector vector, VectorSearchOptions? options = null, CancellationToken cancellationToken = default)
```

Comment:
Do we need to add unit test for this method?

Reply:
This connector doesn't have unit tests for the collection at the moment.  I'm not sure it makes sense to add them at the moment, considering that this connector will need to be replaced by a version that uses the official pinecone client sdk.